core/src/main/resources/defaults/blueprints/7.2.11/cdp-data-engineering-ha.bp

```diff
@@ -552,8 +551,8 @@
         "cardinality": 1,
         "roleConfigGroupsRefNames": [
           "knox-KNOX_GATEWAY-BASE",
-          "hue-HUE_LOAD_BALANCER-BASE",
-          "hue-HUE_SERVER-BASE"
+          "oozie-OOZIE_SERVER-BASE",
+          "spark_on_yarn-GATEWAY-BASE"
```

Comment:
It is very much required. Otherwise the cluster installation was failing. The error message was please install spark_on_yarn gateway in the oozie server group otherwise spark will not work. Something like that which I paraphrased. CC: @fritsi